testsuite/features/secondary/minssh_ansible_control_node.feature

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2021 SUSE LLC
+# Copyright (c) 2021-2022 SUSE LLC
```

Comment:
Ehmmm if we don't change nothing in this file, then it doesn't make sense to update the year.

Reply:
It was changed by someone who forgot to bump the year.